lib/prehog/proto/prehog/v1alpha/teleport.proto

```diff
@@ -0,0 +1,119 @@
+// Copyright 2022 Gravitational, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package prehog.v1alpha;
+
+import "google/protobuf/timestamp.proto";
+
+message UserLoginEvent {
+  // anonymized
+  string user_name = 1;
```

Comment:
```suggestion
  string anonymized_user_name = 1;
```

What do you think about indicating that it's anonymized via the name instead of relying on comments?

Reply:
I do like that, though I think we'd need to change that in the upstream: https://github.com/gravitational/prehog/blob/main/proto/prehog/v1alpha/teleport.proto#L9